File: app/src/main/java/com/ianarbuckle/dublinbushelper/firebase/database/DatabaseHelper.java
package com.ianarbuckle.dublinbushelper.firebase.database;

/**
 * Created by Ian Arbuckle on 04/03/2017.
 *
 */

public interface DatabaseHelper {
  void sendFavouriteStopToDatabase(String time, String stopId, String name, String routes, float lat, float lon);
}

File: app/src/main/java/com/ianarbuckle/dublinbushelper/firebase/database/DatabaseHelper.kt
package com.ianarbuckle.dublinbushelper.firebase.database

import com.ianarbuckle.dublinbushelper.models.Favourites

/**
 * Created by Ian Arbuckle on 04/03/2017.

 */

interface DatabaseHelper {
    fun sendFavouriteStopToDatabase(favourites: Favourites, directory: String)
}
